addons/stock_account/wizard/wizard_valuation_history.py

```diff
@@ -8,23 +8,30 @@ class WizardValuationHistory(models.TransientModel):
     _name = 'wizard.valuation.history'
     _description = 'Wizard that opens the stock valuation history table'
 
-    choose_date = fields.Boolean('Inventory at Date')
-    date = fields.Datetime('Date', default=fields.Datetime.now, required=True)
+    Compute = fields.Selection([
```

Comment:
Wouldn't name a field "compute", and fields shouldn't start with a capital letter.